modules/preview-react/pill/lib/PillIcon.tsx

```diff
@@ -14,17 +14,24 @@ export interface PillIconProps extends Omit<SystemIconProps, 'icon'> {
    * @default `plusIcon`
    */
   icon?: CanvasSystemIcon;
+  /**
+   * The aria label for the icon
+   * @default 'add'
+   */
+  'aria-label'?: string;
 }
 
 export const PillIcon = createSubcomponent('span')({
   modelHook: usePillModel,
-})<PillIconProps>(({size, icon, ...elemProps}, Element) => {
+})<PillIconProps>(({size, icon, 'aria-label': ariaLabel = 'add', ...elemProps}, Element) => {
   return (
     <SystemIcon
       marginInlineStart={`-${space.xxxs}`} // remove padding on the left from HStack
       display="flex"
       as={Element}
       size={20}
+      role="img"
+      aria-label={ariaLabel}
```

Comment:
Small nit, but I think it is easier to understand:

```suggestion
      aria-label="add"
```

So instead of `'aria-label': ariaLabel = 'add'` in the function definition, you remove it. The whole thing looks like this then:

```tsx
export const PillIcon = createSubcomponent('span')({
  modelHook: usePillModel,
})<PillIconProps>(({size, icon, ...elemProps}, Element) => {
  return (
    <SystemIcon
      marginInlineStart={`-${space.xxxs}`} // remove padding on the left from HStack
      display="flex"
      as={Element}
      size={20}
      role="img"
      aria-label="add"
      icon={icon || plusIcon}
      {...elemProps}
    />
  );
});
```